api/pom.xml

```diff
@@ -136,6 +136,11 @@
 			<artifactId>servlet-api</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
```

Comment:
Dont include compile time dependency. OpenMRS already bundles javax.mal https://github.com/openmrs/openmrs-core/blob/master/pom.xml#L362 

So you just need to make the scope as "provided" or "test"

Reply:
Updated the file